module/VuFind/src/VuFindTest/Feature/SearchFacetFilterTrait.php

```diff
@@ -108,4 +116,23 @@ protected function assertAppliedFilter(Element $page, string $expectedFilter): v
         $this->assertEquals('hierarchy:', $label->getText());
         $this->assertEquals("Remove Filter $expectedFilter", $filter->getText());
     }
+
+    /**
+     * Get textual content of a facet element by data-title attribute
```

Comment:
Is this comment accurate? Seems like it takes any selector, so is "by data-title attribute" just an example?

Reply:
Fixed.